js/interface.js

```diff
@@ -57,6 +57,8 @@ var definitionEditor = CodeMirror.fromTextArea($('#definition')[0], {
   mode: 'javascript'
 });
 
+var emptyColumnNameRegex = /Column\s\([0-9]+\)/;
```

Comment:
It's important that this is the entire column name.

```suggestion
var emptyColumnNameRegex = /^Column\s\([0-9]+\)$/;
```

Reply:
Fixed.